lib/hexpm/accounts/session.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Hexpm.Accounts.Session do
```

Comment:
Now I understand what you mean. That's a good point and something we try to follow but we are probably missing it in a few places. @wojtekmach has actually fixed the vast majority of these but old habits are still a bit stuck with me :)